crates/core/tedge_api/src/health.rs

```diff
@@ -1,15 +1,14 @@
 use clock::Clock;
 use clock::WallClock;
+use mqtt_channel::TopicFilter;
 use std::process;
 use std::sync::Arc;
 
```

Comment:
Removing this blank line might help `cargo sort` and prevents `mqtt_channel` imports being split.